src/entities/offerings.ts

```diff
@@ -71,6 +75,57 @@ export interface Price {
   readonly formattedPrice: string;
 }
 
+/**
+ * Represents the price and duration information for a phase of the purchase option.
+ * @public
+ */
+export interface PurchaseOptionPrice {
```

Comment:
Hmm considering this has more than the price, I'm thinking about renaming this to `PurchaseOptionPhase`. Wdyt?